websites/G/GoPlay/metadata.json

```diff
@@ -2,18 +2,22 @@
 	"$schema": "https://schemas.premid.app/metadata/1.11",
 	"apiVersion": 1,
 	"author": {
-		"name": "Striker",
-		"id": "215509157837537280"
+		"name": "Frostware",
+		"id": "1268984565930786899"
 	},
```

Comment:
Since this is just a metadata change, consider leaving the author alone and add yourself as a contributor instead.

Reply:
done